fe/fe-core/src/main/java/com/starrocks/statistic/StatisticUtils.java

```diff
@@ -89,7 +89,7 @@ public class StatisticUtils {
             .add("information_schema").build();
 
     public static ConnectContext buildConnectContext() {
-        ConnectContext context = new ConnectContext();
```

Comment:
please adapt to here. https://github.com/StarRocks/starrocks/blob/main/fe/fe-core/src/main/java/com/starrocks/connector/metadata/MetadataCollectJob.java#L111